network-api/networkapi/donate/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from wagtail.admin.panels import FieldPanel
+from wagtail.fields import RichTextField
+from wagtail.models import Page
+from wagtail_localize.fields import SynchronizedField, TranslatableField
+
+from networkapi.wagtailpages.models import FoundationMetadataPageMixin
+
+
+class DonationPage(FoundationMetadataPageMixin, Page):
+    class Meta:
+        abstract = True
+
+
+class DonateLandingPage(DonationPage):
+    template = "pages/landing_page.html"
+
+    # Only allow creating landing pages at the root level
+    parent_page_types = ["wagtailcore.Page"]
+
+    subpage_types: list = []
```

Comment:
This list is empty for now, but will be updated when we add models such as `CampaignPage` in later tickets.